dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -131,10 +134,17 @@ private FormDataMultiPart createFormDataMultiPart(final String fileName,
     public void Test_Create_app_descriptor_Then_Create_App_Integration_Then_Delete_The_Whole_App() {
 
         final Host host = new SiteDataGen().nextPersisted();
-        final Map<String, Param> paramMap = ImmutableMap.of(
-                "p1", Param.newParam("v1", false, Type.STRING, "label", "hint", true),
-                "p2", Param.newParam("v2", false, Type.STRING, "label", "hint", true),
-                "p3", Param.newParam("v3", false, Type.STRING, "label", "hint", true)
+
+        final SortedMap<String, ParamDescriptor> paramMap = ImmutableSortedMap.of(
+                "p1", ParamDescriptor.newParam("v1", false, Type.STRING, "label", "hint", true),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "label" appears 21 times in this file; the first occurrence is on line 135](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5289044)